tasks/known-hosts.yml

```diff
@@ -16,9 +16,9 @@
     state: directory
     owner: root
     group: root
-    mode: 0755
```

Comment:
Why did you change this?

Reply:
ansible-lint (or yamllint, not sure) complains about hex numbers not in the `0oXXX` format.

Reply:
I prefer to to keep them as they were. I cannot reproduce the linter issues